README.md

```diff
@@ -1,51 +1,48 @@
-### Moya Contributors Guidelines v1.1.0
-
+### Moya Community Continuity Guidelines v2.0.0
 ##### Background
 
-Moya started out as a project in Artsy under the ownership of [Ash Furrow](https://github.com/ashfurrow) and [Orta Therox](https://github.com/orta). Over time, developers from the community began using Moya and it is now a community-driven project.
+Moya started out as a project in Artsy under the ownership of [Ash Furrow](https://github.com/ashfurrow) and [Orta Therox](https://github.com/orta). Over time, developers from the community began using Moya and it became a community-driven project.
 
 After watching [The Social Coding Contract](http://blog.testdouble.com/posts/2014-12-02-the-social-coding-contract.html) we opted to find ways to make [the project](https://github.com/Moya/Moya/issues/135) more welcoming to external contributors.
 
-This document tries to clarify how we can include as many people as possible within the project.
-
-#### How to get push access?
-
-If you get a merged PR, regardless of content (typos, code, doc fixes) then you're eligible for push access to the Moya organization. We check for this on PR merges and extend an invite via GitHub.
-
-Offhand, it's easy to imagine that this would make code quality suffer, but in reality it offers fresh perspectives to the codebase and encourages ownership from people who are depending on the project.
+Because of the created the [Moya Community document](https://github.com/Moya/contributors/blob/0d5e80682b2377bdca72585eda9ce83467bee3c4/README.md) - over time this has been adopted in more projects.
 
-Everyone comes in with their own perspective on what a project could/should look like, and encouraging discussion can help expose good ideas sooner.
+This repo contains a document [Community.md](Community.md) which is a template for others to help foster their own communities. Think of it as a code of conduct for the continuity of your community.
 
-#### When should you use push access?
+---------
 
-It can be overwhelming to suddenly be offered the chance to wipe the source code for a project. Don't worry, we don't let you push to master.
+### I want to apply these guidelines
 
-All code is peer-reviewed, and we have the convention that someone other than the submitter should merge the pull request.
+Great! So, this is not quite another file to copy & paste into your project. It's slightly more involved.
 
-As an org contributor you can merge other people's pull requests, or other contributors can merge yours. You won't be assigned a pull request, but you're welcome to just jump in and take a code review on things that interest you.
+It's not too many steps though, and the majority of them are common sense things you would do anyway.
 
-If it feels right, merge. Moya is not continuously deployed, there is space for debate after too. Offering the chance to revert, or make an amending pull request.
+- [ ] Copy the Changelog.md into your repo
```

Comment:
There is no `CHANGELOG.md` in `Moya/contributors` … so which file am I supposed to copy? 😆 

Didn't you mean `Community.md`?